static/docs/use-cases/shared-storage-on-nfs.md

```diff
@@ -0,0 +1,148 @@
+# Shared Storage on NFS
+
+In the modern software development environment, teams are working together on
+same dataset to get the results. It became necessary that data is accessible and
+every team member has a same updated dataset. NFS (Network File System) storage
+is widely used for storing and sharing files on the network. This allows you to
+have better resource utilization such as ability to store large datasets on a
+single host machine.
+
+With DVC, you can easily setup a shared cache storage on the NFS server that
+will allow your team to share and store data for your projects effectively as
+possible and have a workspace restoration/switching speed as instant as
+`git checkout` for your code.
+
+With large data files it is better to set the cache directory to external NFS.
+Not only just it will cache the data faster but also version the data. Suppose,
+we have a dataset with 1 million images. With DVC, we can have multiple versions
+of a dataset without affecting each other work and without creating duplicates
+of a complete dataset. With `cache directory` set to `NFS server` you would
+avoid copying large files from NFS server to the machine and DVC will manage the
+links from the workspace to cache.
+
```

Comment:
the paragraph above is good but feels repetitive to the first paragraph in the document and has minor problems. What information you are trying to deliver here? Can you summarize it here in the comments, please? And we'll see how can we improve the text.